zebra-consensus/src/parameters/subsidy.rs

```diff
@@ -107,7 +107,7 @@ lazy_static! {
     pub static ref FUNDING_STREAM_HEIGHT_RANGES: HashMap<Network, std::ops::Range<Height>> = {
         let mut hash_map = HashMap::new();
         hash_map.insert(Network::Mainnet, Height(1_046_400)..Height(2_726_400));
-        hash_map.insert(Network::Testnet, Height(1_028_500)..Height(2_796_000));
+        hash_map.insert(Network::new_testnet(), Height(1_028_500)..Height(2_796_000));
```

Comment:
This will cause a panic with a cryptic error message for custom testnets: 
https://github.com/ZcashFoundation/zebra/blob/2efc6bf4b2875f16a987bcbff0f90cff98eda3fa/zebra-consensus/src/block/subsidy/funding_streams.rs#L32